test/Chirp.Infrastructure.Tests/InMemoryDatabaseUnitTest.cs

```diff
@@ -18,11 +18,11 @@ public void MemoryDatabaseShouldNotBeEmpty()
         CheepRepository repository = new(context);
         connection.Open(); //This needs to be last or an error occurs because of the repository being set
 
-        //Act
-        //Check if not empty
+        //Act    Check if the database is empty
+        Action act = () => repository.GetCheeps(1);
 
-        //Assert
-        //If not empty PASS
+        //Assert    If not empty it should PASS
```

Comment:
Two lines are better to keep AAA comments on their own line see below the the three A's
//Arrange 
//Act
//Assert